packages/tooltip/src/Tooltip2D.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
 import Tooltip from './Tooltip';
 
+const useStyles = makeStyles(theme => ({
+  cellEmphasisCrosshair: {
+    zIndex: 50,
+    position: 'absolute',
+    pointerEvents: 'none',
+    boxSizing: 'border-box',
+    backgroundColor: theme.palette.secondaryForeground,
+  },
+}));
+
 /**
```

Comment:
https://github.com/vitessce/vitessce/pull/1333/files#diff-0b3ef692fb0811184095248cb67315ad526334225712f4396750e7a3ffc7d255 just for reference.  Looks identical.